src/nvim/search.c

```diff
@@ -286,6 +286,8 @@ static struct spat saved_last_search_spat;
 static int did_save_last_search_spat = 0;
 static int saved_last_idx = 0;
 static bool saved_no_hlsearch = false;
+static colnr_T saved_search_match_endcol;
+static linenr_T saved_search_match_lines;
```

Comment:
Seems like these could be local to f_searchcount, but I guess module-level is ok.